### PR TITLE
 chore(zh-cn): convert noteblocks for `/learn` (part 1)

### PR DESCRIPTION
**Pull request by PassionPenguin**

### CODE REVIEW

File: files/zh-cn/learn/performance/css/index.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ slug: Learn/Performance/CSS
 
 浏览器可能会在元素实际发生变化之前进行优化设置。这类优化可以通过提前完成可能需要的大量工作，提高页面的响应速度。CSS 的 [`will-change`](/zh-CN/docs/Web/CSS/will-change) 属性向浏览器提示元素预期的变化方式。
 
-> **备注：**`will-change` 应该作为处理现有的性能问题的最后一招。不应该用它来预测性能问题。
+> **备注：** `will-change` 应该作为处理现有的性能问题的最后一招。不应该用它来预测性能问题。
```
Comment (yin1999) [inline files/zh-cn/learn/performance/css/index.md:162]: 撤销吧，这处是能正确渲染的

File: files/zh-cn/learn/css/css_layout/supporting_older_browsers/index.md [hunk 2]
```diff
@@ -92,7 +92,7 @@ CSS 规范包含了在一个物件上同时应用两种布局的时候，解释
 
 {{ EmbedLiveSample('在 CSS 中构建回滚', '100%', '200') }}
 
-> **备注：**{{cssxref("clear")}}属性也在被清除的物件变为网格物件的时候失效，所以你应该在一个会变成网格布局的布局上，加上被清除的尾部。
+> [!NOTE] >{{cssxref("clear")}}属性也在被清除的物件变为网格物件的时候失效，所以你应该在一个会变成网格布局的布局上，加上被清除的尾部。
```
Comment (yin1999) [inline files/zh-cn/learn/css/css_layout/supporting_older_browsers/index.md:95]:
```suggestion
> **备注：** {{cssxref("clear")}}属性也在被清除的物件变为网格物件的时候失效，所以你应该在一个会变成网格布局的布局上，加上被清除的尾部。
```

File: files/zh-cn/learn/accessibility/wai-aria_basics/index.md [hunk 3]
```diff
@@ -246,9 +250,10 @@ var intervalID = window.setInterval(showQuote, 10000);
 
 这个 `aria-atomic="true"` 属性告诉屏幕阅读器去读取整个元素的内容作为一个原子单位，而不是里头某个字符串更新了。
 
-> **备注：** 你可在此完成此例子 [aria-live.html](https://github.com/mdn/learning-area/blob/main/accessibility/aria/aria-live.html) ([在线 demo](http://mdn.github.io/learning-area/accessibility/aria/aria-live.html)).
+> [!NOTE]
+> 你可在此完成此例子 [aria-live.html](https://github.com/mdn/learning-area/blob/main/accessibility/aria/aria-live.html) ([在线 demo](http://mdn.github.io/learning-area/accessibility/aria/aria-live.html)).
 
-> **备注：** [`aria-relevant`](https://www.w3.org/TR/wai-aria-1.1/#aria-relevant) 属性对于控制更新实时区域时读取的内容也非常有用。例如，你读取内容添加或删除。
+> [!NOTE] > [`aria-relevant`](https://www.w3.org/TR/wai-aria-1.1/#aria-relevant) 属性对于控制更新实时区域时读取的内容也非常有用。例如，你读取内容添加或删除。
```
Comment (yin1999) [inline files/zh-cn/learn/accessibility/wai-aria_basics/index.md:256]:
```suggestion
> **备注：** [`aria-relevant`](https://www.w3.org/TR/wai-aria-1.1/#aria-relevant) 属性对于控制更新实时区域时读取的内容也非常有用。例如，你读取内容添加或删除。
```

File: files/zh-cn/learn/accessibility/css_and_javascript/index.md [hunk 4]
```diff
@@ -204,7 +205,7 @@ a:active {
 
 另一方面，不应使用 {{cssxref("visibility")}}`:hidden` 或 {{cssxref("display")}}`:none`，因为它们会隐藏屏幕阅读器中的内容。当然，除非你希望从屏幕阅读器中隐藏此内容，这是有充分理由的。
 
-> **备注：** [专为屏幕阅读器用户设计的不可见内容](http://webaim.org/techniques/css/invisiblecontent/)有围绕本主题的更多有用详细信息。
+> [!NOTE] > [专为屏幕阅读器用户设计的不可见内容](http://webaim.org/techniques/css/invisiblecontent/)有围绕本主题的更多有用详细信息。
```
Comment (yin1999) [inline files/zh-cn/learn/accessibility/css_and_javascript/index.md:208]:
```suggestion
> **备注：** [专为屏幕阅读器用户设计的不可见内容](http://webaim.org/techniques/css/invisiblecontent/)有围绕本主题的更多有用详细信息。
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/learn/performance/css/index.md (hunk 1 above)
Comment (PassionPenguin) [reply files/zh-cn/learn/performance/css/index.md:162]:
```suggestion
> **备注：**`will-change` 应该作为处理现有的性能问题的最后一招。不应该用它来预测性能问题。
```

File: files/zh-cn/learn/css/css_layout/supporting_older_browsers/index.md [hunk 5]
```diff
@@ -92,7 +92,7 @@ CSS 规范包含了在一个物件上同时应用两种布局的时候，解释
 
 {{ EmbedLiveSample('在 CSS 中构建回滚', '100%', '200') }}
 
-> **备注：**{{cssxref("clear")}}属性也在被清除的物件变为网格物件的时候失效，所以你应该在一个会变成网格布局的布局上，加上被清除的尾部。
+> **备注：** {{cssxref("clear")}}属性也在被清除的物件变为网格物件的时候失效，所以你应该在一个会变成网格布局的布局上，加上被清除的尾部。
```
Comment (yin1999) [inline files/zh-cn/learn/css/css_layout/supporting_older_browsers/index.md:95]:
```suggestion
> **备注：**{{cssxref("clear")}}属性也在被清除的物件变为网格物件的时候失效，所以你应该在一个会变成网格布局的布局上，加上被清除的尾部。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**